assets/views/home/references.ejs

```diff
@@ -1,5 +1,15 @@
 <div class="row">
   <table class="table" id="referencesTable">
+    <thead>
+      <tr class="no-space">
+        <th class="col-xs-1 flair-number"></th>
```

Comment:
This seems like a weird way to force a width on a column, adding extra elements.

Could we not do

```
tr td:first-child {
  width: 50px;
}
```

Which should fix the width on each of the first cells on each row? Might be nicer.

Reply:
Tried that. It doesn't work.

![https://i.imgur.com/h7yfvie.png](https://i.imgur.com/h7yfvie.png)

Anything else I tried would break something else.

Reply:
Oh yes, because that would do it for the titles as well. 

Just to humour me, try

```
tr td:first-child:not(.firstHeading) {
  width: 1000px;
}
```

If it doesn't work, that's fine, I'll approve this.

Reply:
Ah wait no, of course that won't work, because it only applies to the first. Right, fine, have your approval, but only because this html is already a crapshow.